src/core_plugins/tag_cloud/public/cloud_directive.js

```diff
@@ -0,0 +1,45 @@
+import _ from 'lodash';
+import uiModules from 'ui/modules';
+import TagCloud from 'plugins/tagcloud/tag_cloud';
+
+const module = uiModules.get('kibana/tagcloud', ['kibana']);
+
+module.directive('kbnTagCloud', function () {
+
+  return {
+    restrict: 'E',
+    scope: {
+      data: '=',
+      options: '='
+    },
+    template: '<svg class="parent"></svg>',
+    replace: 'true',
+    link: function (scope, element) {
+
+      const tagCloud = new TagCloud(element[0], containerSize());
+
+      function containerSize() {
+        return [element.parent().width(), element.parent().height()];
```

Comment:
For explicitness can we rename this function to use a verb, and return an object?

```javascript
function getContainerSize() {
  return {
    width: element.parent().width(),
    height: element.parent().height(),
  }
}
```